### DISCUSSION TITLE
Attempt to read property "name" on null

### DISCUSSION THREAD

**Discussion by rajibgit** [livewire/livewire#4797] [1. Help]
I can not get any clue to fix this issues. How can I go forward. Someone help me please.
![image](https://user-images.githubusercontent.com/36604932/157999299-0b451be4-2cc1-481e-81d7-18e55893f705.png)

**Comment by joshhanley** [comment]
@rajibgit could you share your code which reproduces the issue with as little code as possible so we can investigate it? As well as a screenshot that shows the full page of the stack trace? Thanks!
